src/VisualLinter/Linting/LinterResults.cs

```diff
@@ -0,0 +1,27 @@
+﻿using jwldnr.VisualLinter.Enums;
+using System.Collections.Generic;
+
+namespace jwldnr.VisualLinter.Linting
+{
+    public class LinterResults : List<LinterResult>
+    {
+        public LinterType LinterType { get; set; }
+
+        public LinterResults()
+        {
+            this.LinterType = LinterType.Unknown;
+        }
+
+        public LinterResults(EslintResults eslintResults)
```

Comment:
Try splitting these different constructor calls to separate classes (see above comments for reasoning)